### ISSUE TITLE
About training loss on my custom datasets

### ISSUE THREAD

**Issue by QUTGXX** [vskadandale/vocalist#8]
Hi, thanks for sharing this excellent work. I want to train this work on my own dataset, Can u tell me how the loss changes during training and testing?   I'm looking forward to your suggestions at your earliest convenience.
